src/app-bar.jsx

```diff
@@ -260,7 +268,7 @@ const AppBar = React.createClass({
         }
 
         menuElementLeft = (
-          <div style={prepareStyles(Object.assign({}, styles.iconButtonStyle))}>
+          <div style={prepareStyles(iconLeftStyle)}>
```

Comment:
Oh, I'm sorry. Including a test, and re-commit again at a later date. :dizzy_face: